guides/doc-Provisioning_Guide/topics/Virt-Libvirt.adoc

```diff
@@ -150,6 +161,8 @@ Create the connection with the `hammer compute-resource create` command:
 
 If you want to use image-based provisioning to create hosts, you must add information about the image to your {ProjectServer}. This includes access details and the image location.
 
+NOTE: Only directory pool storage types can be managed via {ProjectX}.
```

Comment:
You're adding in content again rather than just marking up the existing content! I will add a bug to add this to the downstream. Just try not to do this in future as part of this. Add it as an issue to be fixed by me, or another PR.

Reply:
Going to split the commit into two so you can easily cherry pick downstream and it's cleaner. Agreed.